stdlib/public/Concurrency/Task.swift

```diff
@@ -186,6 +186,42 @@ extension Task: Equatable {
   }
 }
 
+@available(SwiftStdlib 5.9, *)
+extension Task {
+    @_spi(MainActorUtilities)
+    @MainActor
+    public static func startOnMainActor(
+        priority: TaskPriority? = nil,
+        _ work: @escaping @MainActor() async throws -> Success
+    ) -> Task<Success, Error> {
+        let flags = taskCreateFlags(priority: priority, isChildTask: false,
+                                    copyTaskLocals: false, inheritContext: true,
+                                    enqueueJob: false,
+                                    addPendingGroupTaskUnconditionally: false)
+        let (task, _) = Builtin.createAsyncTask(flags, work)
+        _startTaskOnMainActor(task)
+        return Task<Success, Error>(task)
+    }
+}
+
+@available(SwiftStdlib 5.9, *)
+extension Task where Failure == Never {
+    @_spi(MainActorUtilities)
+    @MainActor
+    public static func startOnMainActor(
+        priority: TaskPriority? = nil,
+        _ work: @escaping @MainActor() async -> Success
+    ) -> Task<Success, Never> {
+        let flags = taskCreateFlags(priority: priority, isChildTask: false,
+                                    copyTaskLocals: false, inheritContext: true,
```

Comment:
This also should be true
```suggestion
                                    copyTaskLocals: true, inheritContext: true,
```